java/amazon-kinesis-producer/src/main/java/com/amazonaws/services/kinesis/producer/BinaryToHexConverter.java

```diff
@@ -0,0 +1,28 @@
+package com.amazonaws.services.kinesis.producer;
+
+public class BinaryToHexConverter {
+
+	private static final BinaryToHexConverter INSTANCE = new BinaryToHexConverter();
+	private static final char[] HEX_CODE = "0123456789ABCDEF".toCharArray();
+
+	/**
+	 * Converts an array of bytes into a hex string.
+	 *
+	 * @param data An array of bytes
+	 * @return A string containing a lexical representation of xsd:hexBinary
+	 * @throws IllegalArgumentException if {@code data} is null.
+	 */
+	public static String convert(byte[] data) {
+		return INSTANCE.convertToHex(data);
+	}
+
+	private String convertToHex(byte[] data) {
```

Comment:
The implementation here is the same as in the original jaxb `DatatypeConverter.printHexBinary`.